static/src/javascripts/projects/common/modules/email/email-article.js

```diff
@@ -138,10 +140,17 @@ define([
         },
         listCanRun = function (listConfig) {
             // Check our lists canRun method and make sure that the user isn't already subscribed to this email
-            if (listConfig.canRun && canRunList[listConfig.canRun]() && !contains(userListSubscriptions, listConfig.listId)) {
+            if (listConfig.canRun &&
```

Comment:
canRun is a misleading name (I know it's existing code) but I think this is more like listName?